src/Resume/Resume.jsx

```diff
@@ -0,0 +1,185 @@
+              <div>{info.skillsCommunication}</div>
+              <div>{info.skillsAnalytical}</div>
+              <div>{info.skillsResponsibility}</div>
+              <div>{info.skillsPurpose}</div>
+          </div>
+          <hr className='line_left'></hr>
+
+          <div className="info">
+              <p className="heading">{info.languages}</p>
+              <div className='rating_home'>  
+                <div className='ger give_width'>
+                  <span className='after_icon'>{info.langGerm}</span>
+                  </div>
+                <div> 
+                  <img className='rating_star' src={rating_4} alt="4 stars" />
+                </div>
+              </div>
+              <div className='rating_home'>  
+                <div className='eng give_width'>
+                <span className='after_icon'>{info.langEng}</span></div>
+                <div> 
+                  <img className='rating_star' src={rating_4} alt="4 stars" />
+                </div>
+              </div>
+
+              <div className='rating_home'>  
+                <div className='rus give_width'>
+                <span className='after_icon'>{info.langRus}</span></div>
+                <div> 
+                  <img className='rating_star' src={rating_5} alt="5 stars" />
+                </div>
+              </div>
+              <div className='rating_home'>  
+                <div className='heb give_width'>
+                <span className='after_icon'>{info.langHeb}</span>
+                  </div>
+                <div> 
+                  <img className='rating_star' src={rating_5} alt="5 stars" />
+                </div>
+              </div>
+
+          </div>
+          <hr className='line_left'></hr>
+      </div>
+
+    <div className='description'> 
+    <button className='btn btn-ger' onClick={goToGermanHandler}> </button>
+    <button className='btn btn-eng' onClick={goToEnglishHandler}></button>
+      <div className="experience">
+        <p className='heading heading_prof'>
+           {info.motivationTitle}
+        </p>
+         
+          <hr className='line'></hr>
+          <Topic description={info.motivationText}/>
+          
+        <div className='experience'> 
+          <p className='heading heading_prof'>Professional Experience </p></div>
+          <hr className='line'></hr>
+          <Topic organization={info.BiProfOrg} title = {info.BiProfTitle} description ={info.BiProfDesc}/>
+          <Topic organization={info.GenProfOrg} title = {info.GenProfTitle} description ={info.GenProfDesc}/>
+          <Topic organization={info.JafiProfOrg} title = {info.JafiProfTitle} description ={info.JafiProfDesc}/>
+         
+          <div className='experience'> 
+          <p className='heading heading_prof'>{info.professionalCourses}</p></div>
+          <hr className='line'></hr>
+          
+          <p className='heading_prof heading_org'>{info.courses}</p>
+            <div className='exp-text'>
```

Comment:
Das kann man auch in eine eingene Komponente reinpacken.

```
<div className='exp-text'> 
   <img className='arrow_icon' src={right_arrow} alt="arrow" />
   <span>{info.courseWeb}</span>
</div>
```

Reply:
Habe gemacht. CourseItem Component wurde geschrieben